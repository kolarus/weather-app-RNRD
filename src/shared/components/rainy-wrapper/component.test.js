import React from 'react';
import {render} from '@testing-library/react-native';

import RainyWrapper from './component';

test('RainyWrapper should match snapshot', () => {
  const {toJSON} = render(<RainyWrapper />);

  expect(toJSON()).toMatchSnapshot();
});
