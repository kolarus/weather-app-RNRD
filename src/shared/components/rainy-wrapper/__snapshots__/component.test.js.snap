// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RainyWrapper should match snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#393c50",
        "flexDirection": "row",
        "height": "100%",
        "justifyContent": "center",
      },
      undefined,
    ]
  }
>
  <View
    accessibilityIgnoresInvertColors={true}
    style={
      Object {
        "bottom": 0,
        "justifyContent": "flex-start",
        "left": 0,
        "opacity": 0.1,
        "position": "absolute",
        "resizeMode": "cover",
        "right": 0,
        "top": 0,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/shared/components/rainy-wrapper/rain-drops.png",
        }
      }
      style={
        Array [
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          },
          Object {
            "height": undefined,
            "width": undefined,
          },
          undefined,
        ]
      }
    />
  </View>
  <RNCSafeAreaView />
</View>
`;
